searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp

```diff
@@ -929,57 +929,6 @@ MaintenanceControllerFixture::removeDocs(const test::UserDocuments &docs, Timest
     }
 }
 
-TEST_F("require that bucket move controller is active", MaintenanceControllerFixture)
```

Comment:
@baldersheim Is a similar test relevant for bucket mover v2?

Reply:
The bucket move job is tested thoroughly in documentbucketmover_test.cpp. So I think this one is not necessary.